src/services/cleanFormSubmissionModel.ts

```diff
@@ -219,16 +219,16 @@ function cleanElementValue(
       default: {
         if (!formElementsConditionallyShown?.[element.name]?.isHidden) {
           model[element.name] = submission[element.name]
-        }
-        switch (element.type) {
-          case 'text':
-          case 'textarea':
-          case 'email':
-          case 'barcodeScanner':
-          case 'telephone': {
-            const value = submission[element.name] as string
-            model[element.name] = value.trim()
-            break
+          switch (element.type) {
+            case 'text':
+            case 'textarea':
+            case 'email':
+            case 'barcodeScanner':
+            case 'telephone': {
+              const value = submission[element.name] as string
```

Comment:
Why not declare this line before the switch, and then use a default case to catch all the other elements to do

```
model[element.name] = submission[element.name]
```
Instead of assigning it twice?